third_party/blink/renderer/modules/cobalt/h5vcc_experiments/experiments_utils.cc

```diff
@@ -0,0 +1,68 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "third_party/blink/renderer/modules/cobalt/h5vcc_experiments/experiments_utils.h"
+
+#include "third_party/blink/renderer/bindings/core/v8/v8_union_boolean_long_string.h"
+
+namespace blink {
+
+std::optional<base::Value::Dict> ToBaseValueDict(
```

Comment:
How about this method is just something like "ParseConfigToDictionary" or something like that? The fact it's a "base value dict" seems like an implementation detail.